libs/eo/shared/services/src/idle-timer/idle-timer.service.ts

```diff
@@ -102,14 +96,9 @@ export class IdleTimerService {
 
     if (!lastActivity) lastActivity = new Date().getTime();
 
-    const countdown = timer(0, 1000).pipe(
+    const countdown$ = timer(0, 1000).pipe(
       takeUntil(this.destroy$),
```

Comment:
Suggestion: Use takeUntilDestoryed

Reply:
It does not work like a destroy$ normally does, I've renamed the subject to avoid confusion.